tests/byzantium/eip198_modexp_precompile/test_modexp.py

```diff
@@ -72,6 +72,10 @@
             ModExpInput(base="", exponent="", modulus="0001"),
             ModExpOutput(returned_data="0x0000"),
         ),
+        (
+            ModExpInput(base="", exponent="", modulus="", declared_exponent_length=2**32, declared_modulus_length=1),
```

Comment:
Placing the test here doesn't work because the test implementation cannot allocate an account or something.